drivers/SmartThings/matter-sensor/capabilities/customRainSensor.yml

```diff
@@ -0,0 +1,18 @@
+name: Custom Rain Sensor
+id: smilevirtual57983.customRainSensor
```

Comment:
We can use the [rainSensor](https://github.ecodesamsung.com/iot/SmartThingsCapabilities/blob/main/capabilities/rain-sensor.src/rain-sensor.yml) capability. right?

Reply:
at this point, I think we can? Not 100% sure.

Reply:
[rainSensor](https://github.ecodesamsung.com/iot/SmartThingsCapabilities/blob/main/capabilities/rain-sensor.src/rain-sensor.yml) was merged to production branch yesterday.